src/scenes/home/ListTasks.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import { FlatList } from 'react-native'
+import { NavigationRoute, NavigationScreenProp } from 'react-navigation'
+
+import { Task } from 'src/scenes/home/Task'
+import { Task as typeTask } from 'src/types/global'
+
+export interface Props {
+  navigation: NavigationScreenProp<NavigationRoute>
+  tasks: typeTask[]
+}
+
+class ListTasks extends Component<Props> {
```

Comment:
`export class ListTasks extends Component<Props> {`
don't use export default, always try to use export const